node/src/crypto/error.rs

```diff
@@ -1,6 +1,7 @@
 use std::result;
 
 use base64::DecodeError;
+use ed25519_dalek::ed25519::Error as SignatureError;
```

Comment:
```suggestion
use signature::Error as SignatureError;
```

I _think_ that's what we want here.

Reply:
Suggestion applied